<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>510</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Big Label</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>1041</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_sampling">
      <property name="text">
       <string>Edit Samplings</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_crossover">
      <property name="text">
       <string>Edit Crossovers</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_mutation">
      <property name="text">
       <string>Edit Mutations</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_selection">
      <property name="text">
       <string>Edit Selections</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_decomposition">
      <property name="text">
       <string>Edit Decompositions</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ref_dirs">
      <property name="text">
       <string>Edit Reference Directions</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Defaults&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="variants_table">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>1041</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>79</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Object ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Class</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg 2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg 3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg 5</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg 6</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg 7</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="defaults_table">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>1041</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>79</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Object ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Class</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg 2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg 3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg 5</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg 6</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Arg 7</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Variants&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox_2">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>750</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
